cri/v1alpha2/cri.go

```diff
@@ -273,11 +274,14 @@ func (c *CriManager) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 		}
 	}
 
+	enableLxcfs, _ := strconv.ParseBool(config.Annotations[anno.LxcfsEnabled])
```

Comment:
What will happen if `config.Annotations[anno.LxcfsEnabled]` is invalid and the parsing returns a non-nil error? @Starnop